examples/zipkin/src/main/java/io/opentelemetry/example/zipkin/ExampleConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.example.zipkin;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.exporter.zipkin.ZipkinSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SdkTracerManagement;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+
+/**
+ * All SDK management takes place here, away from the instrumentation code, which should only access
+ * the OpenTelemetry APIs.
+ */
+public class ExampleConfiguration {
+  // Zipkin API Endpoints for uploading spans
+  private static final String ENDPOINT_V2_SPANS = "/api/v2/spans";
+
+  // Name of the service
+  private static final String SERVICE_NAME = "myExampleService";
+
+  // SDK Tracer Management interface
+  private static SdkTracerManagement sdkTracerManagement;
+
+  // This method adds SimpleSpanProcessor initialized with ZipkinSpanExporter to the
+  // TracerSdkProvider
+  static OpenTelemetry initializeOpenTelemetry(String ip, int port) {
+    OpenTelemetrySdk openTelemetrySdk = OpenTelemetrySdk.builder().build();
+
+    String httpUrl = String.format("http://%s:%s", ip, port);
+    ZipkinSpanExporter zipkinExporter =
+        ZipkinSpanExporter.builder()
+            .setEndpoint(httpUrl + ENDPOINT_V2_SPANS)
+            .setServiceName(SERVICE_NAME)
+            .build();
+    // save the management interface so we can shut it down at the end of the example.
+    sdkTracerManagement = openTelemetrySdk.getTracerManagement();
+    // Set to process the spans by the Zipkin Exporter
+    sdkTracerManagement.addSpanProcessor(SimpleSpanProcessor.builder(zipkinExporter).build());
```

Comment:
This separation looks nice, and I suspect we should go for it. But it'd be best if we can update to the new `Builder.addSpanProcessor.buildAndRegisterGlobal()` type of pattern to confirm what the end result looks like.

Reply:
We can't do that until 0.14.0, correct? The examples are still using 0.13.1 for now, until we release.

Reply:
I forget what's in what version 😅  Yeah I think so.

Reply:
ok, updated for 0.14.1!